docs/TELEMETRY.md

```diff
@@ -0,0 +1,119 @@
+# Telemetry system
```

Comment:
Is there a link to Telemetry docs in root `README.md`?

Reply:
No, we don't have it for `NOTIFICATION.md` either. Added new bug #2941 to resolve this.